rpc/backend/tracing_test.go

```
@@ -3,55 +3,53 @@ package backend
 import (
 	"fmt"
 
-	tmlog "cosmossdk.io/log"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+
 	abci "github.com/cometbft/cometbft/abci/types"
 	tmrpctypes "github.com/cometbft/cometbft/rpc/core/types"
 	"github.com/cometbft/cometbft/types"
-	dbm "github.com/cosmos/cosmos-db"
-	"github.com/cosmos/cosmos-sdk/crypto"
-	"github.com/ethereum/go-ethereum/common"
-	ethtypes "github.com/ethereum/go-ethereum/core/types"
-	"github.com/zeta-chain/ethermint/crypto/ethsecp256k1"
-	"github.com/zeta-chain/ethermint/indexer"
-	evmtypes "github.com/zeta-chain/ethermint/x/evm/types"
-	rpctypes "github.com/zeta-chain/node/rpc/types"
 
+	dbm "github.com/cosmos/cosmos-db"
+	"github.com/cosmos/evm/crypto/ethsecp256k1"
+	"github.com/cosmos/evm/indexer"
+	evmtypes "github.com/cosmos/evm/x/vm/types"
 	"github.com/zeta-chain/node/rpc/backend/mocks"
+
+	"cosmossdk.io/log"
+
+	"github.com/cosmos/cosmos-sdk/crypto"
 )
 
-func (suite *BackendTestSuite) TestTraceTransaction() {
-	msgEthereumTx, _ := suite.buildEthereumTx()
-	msgEthereumTx2, _ := suite.buildEthereumTx()
+func (s *TestSuite) TestTraceTransaction() {
```

Again, why ignoring the error in this file?